examples/zkapps/06-offchain-storage/experimental-zkapp-offchain-storage/README.md

```diff
@@ -16,30 +16,30 @@ npm install
 
 #### Blockchain Mode
 
-In the `main.ts` file, you can toggle the `useLocalBlockchain` variable to switch between local testing and real-world usage.
+In the `main.ts` file, you can toggle the `useLocalBlockchain` variable to switch between local simulated blockchain testing and real-world usage.
```

Comment:
Are there any other words we can use instead of simulated?

Reply:
With Lightnet blockchain, we needed a way to describe the simulated blockchain

see the updated docs! 
https://docs.minaprotocol.com/zkapps/tutorials/hello-world#simulated-local-blockchain
https://docs.minaprotocol.com/zkapps/testing-zkapps-locally#creating-a-simulated-local-blockchain

you inspired me to add this term to the glossary, please review (in entirety, for learning, if you like!) the glossary, esp in PR https://github.com/o1-labs/docs2/pull/836